config/samples/dataplane_v1beta1_openstackdataplane_baremetal.yaml

```diff
@@ -64,7 +64,7 @@ spec:
             nova_libvirt_network: internal_api
           edpm_chrony_ntp_servers:
             - clock.redhat.com
-          growvols_args: '/var=100%'
+          growvols_args: '/var=80%'
```

Comment:
Let's just remove it from here and fix the default in edpm-ansible role. We don't need to do it in the sample now that we kustomize in install_yamls and the current tuning is our CI specific.

Reply:
100% agree with @rabi here. Let me propose the pr.

Reply:
Opened https://github.com/openstack-k8s-operators/edpm-ansible/pull/200 but slightly changed approach.